app/src/main/java/io/appium/uiautomator2/model/settings/IncludeA11yActionsInPageSource.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.uiautomator2.model.settings;
+
+public class IncludeA11yActionsInPageSource extends AbstractSetting<Boolean> {
+    private static final String SETTING_NAME = "includeA11yActionsInPageSource";
```

Comment:
Please also do not forget this new setting into settings documentation after this PR is released